.travis.yml

```diff
@@ -7,15 +6,15 @@ env:
         - REPO=multiarch/qemu-user-static
         - PACKAGE_URI="https://kojipkgs.fedoraproject.org/packages/qemu/3.1.0/2.fc30/x86_64/qemu-user-static-3.1.0-2.fc30.x86_64.rpm"
         - PACKAGE_FILENAME=qemu-user-static-3.1.0-2.fc30.x86_64.rpm
-branches:
-  only:
-    - master
+#branches:
+#  only:
+#    - master
 before_script:
     - sudo apt-get install jq rpm2cpio cpio
     - wget --content-disposition $PACKAGE_URI
-    - sudo rpm2cpio $PACKAGE_FILENAME | cpio -dimv
+    - rpm2cpio $PACKAGE_FILENAME | cpio -dimv
 script:
-    - if [[ $TRAVIS_BRANCH == 'master' && $TRAVIS_PULL_REQUEST == 'false' ]]; then sudo ./publish.sh -v "$VERSION" -t "$GITHUB_TOKEN" -r "$REPO"; fi
-    - if [[ $TRAVIS_BRANCH == 'master' && $TRAVIS_PULL_REQUEST == 'false' ]]; then sudo ./update.sh -v "$VERSION" -r "$REPO"; fi
```

Comment:
I think it's should remain outside.

Reply:
I now split the tarball generation to a separate script and I moved the condition outside.